### ISSUE TITLE
Success window for dependency tool keeps appearing

### ISSUE THREAD

**Issue by davisein** [vertexclique/tokamak#28]
I am getting this notification every time I start atom.

I can see why this is useful for development of the plugin or if you were struggling to get it working (the first times I was missing racer), but after that it gets annoying specially if you use atom for more than Rust.

![screen shot 2016-04-14 at 15 55 08](https://cloud.githubusercontent.com/assets/1065966/14530436/4cd95eec-0259-11e6-8c79-e00acd51284f.png)

Can we remove it or make it more intelligent?

**Comment by vertexclique** [comment]
If you are mentioning toolchain detection code then I will refactor it.
This will show once after dependency installation complete.

**Comment by vertexclique** [comment]
Actually it shows up only if you have missing package and then it tries to install thereafter shows this up.

**Comment by davisein** [comment]
@vertexclique It seems to show once for every Window open. I have tried closing Atom and reopening it and it was shown again. Can I help somehow to diagnose?

I have tried closing atom and reopening again 3 times. Each time it showed the same message even without the `Detect binaries on startup` being enabled. I'm using 0.2.7 version of Tokomak on Atom 1.7.1

Let me know how can I be of help and thanks for the good work.
